math/mathmore/src/GSLNLSMinimizer.cxx

```diff
@@ -202,31 +220,36 @@ bool GSLNLSMinimizer::Minimize() {
    std::vector<double> startValues;
 
    // transformation need a grad function.
-   std::unique_ptr<MultiNumGradFunction> gradFunction(new MultiNumGradFunction(*fChi2Func));
-   std::unique_ptr<MinimTransformFunction> trFuncRaw(CreateTransformation(startValues, gradFunction.get()));
+   std::unique_ptr<MultiNumGradFunction> gradFunction;
+   std::unique_ptr<MinimTransformFunction> trFuncRaw;
+   if (!fUseGradFunction) {
+      gradFunction.reset(new MultiNumGradFunction(fitFunc));
```

Comment:
```suggestion
      gradFunction = std::make_unique<MultiNumGradFunction>(fitFunc);
```
Now that the minimum standard is C++14, we should go for `make_unique` in these cases